docs/docs/User Guide/Deployment/High Availability.md

```
@@ -147,20 +147,27 @@ aws s3api put-bucket-versioning \
   --versioning-configuration Status=Enabled
 ```
 
-**MinIO:**
+**Garage:**
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)